src/rebuild/rebuild_iv.c

```diff
@@ -186,12 +186,12 @@ rebuild_iv_ent_refresh(struct ds_iv_entry *entry, struct ds_iv_key *key,
 			return 0;
 		}
 
-		D_DEBUG(DB_REBUILD, DF_UUID"/%u/%u rebuild status gsd/gd %d/%d"
-			" stable eph "DF_U64" resync ver %u\n",
-			DP_UUID(src_iv->riv_pool_uuid), src_iv->riv_ver,
-			src_iv->riv_rebuild_gen, dst_iv->riv_global_scan_done,
-			dst_iv->riv_global_done, dst_iv->riv_stable_epoch,
-			dst_iv->riv_dtx_resyc_version);
+		D_DEBUG(DB_REBUILD, DF_UUID"/%u/%u/"DF_U64" rebuild status gsd/gd %d/%d"
+			" stable eph "DF_X64" resync ver %u\n",
+ 			DP_UUID(src_iv->riv_pool_uuid), src_iv->riv_ver,
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs